src/stable/lib/globals.ts

```diff
@@ -12,42 +12,51 @@ import { Action } from './state';
 
 declare global {
     // eslint-disable-next-line no-var
-    var _azleActions: Action[];
+    var _azleActions: Action[] | undefined;
     // eslint-disable-next-line no-var
     var _azleCanisterClassMeta: CanisterClassMeta | undefined;
     // eslint-disable-next-line no-var
-    var _azleCanisterMethodNames: { [key: string]: string };
+    var _azleCanisterMethodNames: { [key: string]: string } | undefined;
```

Comment:
In src/experimental/build/commands/build/wasm_binary/rust/experimental_canister_template/src/benchmarking.rs when we do .get(functionName) I'm guessing that if this is undefined the return from that get will be None? Is that okay here? Or do we want to check if it's undefined as that is a different error than if the function name isn't in the list? Or do we not care since it's experimental?

Reply:
I saw we're good especially since it's experimental, it will throw an error